.gitignore

```diff
@@ -257,18 +257,6 @@ cython_debug/
 # .nfs files are created when an open file is removed but is still being accessed
 .nfs*
 
-### VirtualEnv template
-# Virtualenv
-# http://iamzed.com/2009/05/07/a-primer-on-virtualenv/
-[Bb]in
```

Comment:
Awesome. On a recent PR I also moved a couple scripts into `/scripts` and deleted these lines from `.gitignore` . Maybe that creates a small merge conflict, not a problem though.